include/mbedtls/config.h

```diff
@@ -1685,6 +1685,28 @@
  */
 #define MBEDTLS_SSL_MAX_FRAGMENT_LENGTH
 
+/**
+ * \def MBEDTLS_SSL_HS_DEFRAG_CLT
+ *
+ * Allows the client to reassemble fragmented TLS handshake messages.
+ *
+ * Requires: MBEDTLS_MPS_READER_C
+ *
+ */
+#define MBEDTLS_SSL_HS_DEFRAG_CLT
```

Comment:
I understand that we're only interested in client-side reassembly, but the implementation works equally well on the server-side, doesn't it?

Reply:
I'd prefer a more intuitive option name. How about `MBEDTLS_SSL_TLS_HANDSHAKE_REASSEMBLY`?

Reply:
I agree that the option name should not be specific to the client. And I have no objections to a more intuitive option name.

One caveat is that I have not tested the server side support yet.